src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -98,6 +98,8 @@ private slots:
     // void breaksManual() { mxmlIoTestRefBreaks("testBreaksManual"); } fail after sync with 3.x
     // void breaksPage() { mxmlMscxExportTestRefBreaks("testBreaksPage"); } fail after sync with 3.x
     // void breaksSystem() { mxmlMscxExportTestRefBreaks("testBreaksSystem"); } fail after sync with 3.x
+    void changeTranspose() { mxmlIoTest("testChangeTranspose"); }
+    void changeTransposeNoDiatonic() { mxmlIoTestRef("testChangeTranspose-no-diatonic"); }
```

Comment:
This test however fails. If I import `testChangeTranspose-no-diatonic.xml` and export it again (using the -d option and no formatting export in preferences, as apparently these tests do too) I don't see and differences between that export and `testChangeTranspose-no-diatonic-ref.xml`.

Reply:
Acrually I checked it with `vim`'s diff. The 'real' diff shows the problem: DOS line endings in the ref. file, UNIX line endings in the exported one. Is that maybe the problem?

Reply:
White-space characters have bitten me before when mixing Linux and Windows diff ...

Reply:
AHRGL!!! I'm just too stupid, the file name of the ref file was wrong, `...-ref.xml` rather than `..._ref.xml`!